Sources/UI/Chat/Chat View Controller/ChatViewController.swift

```diff
@@ -308,6 +308,19 @@ open class ChatViewController: ViewController, UITableViewDataSource, UITableVie
         defaultActionsContextMenu(from: cell, for: message, locationInView: locationInView)
     }
     
+    /// Creates a chat view controller for the message being replied to.
+    ///
+    /// - Parameters:
+    ///     - channelPresenter: the channel presenter of the message being replied to.
+    /// - Returns: a chat view controller.
+    open func createChatViewController(with channelPresenter: ChannelPresenter) -> ChatViewController {
```

Comment:
Can you rename this to `createThreadChatViewController` or something along the lines of this? Having `createChatViewController` under `ChatViewController` is strange